src/anjaliS/IsFibonacciSeriesorNot.java

```diff
@@ -0,0 +1,36 @@
+package anjaliS;
+
+import java.util.Scanner;
+
+public class IsFibonacciSeriesorNot {
+	void displayIsFibonacci(int n) {
+		int firstnumber = 0;
+		int secondnumber = 1;
+		for (int i = 0; i < n; i++) {
+			int sum = firstnumber + secondnumber;
+			firstnumber = secondnumber;
+			secondnumber = sum;
+			if (n == sum) {
+				System.out.println(+n + " is a fibonacciSeries");
+				break;
+			} else if (sum > n) {
```

Comment:
We should not write else condition here, think when first time you are executing it will go into else condition & break the loop,